include/aspect/mesh_deformation/fastscapecc_adapter.h

```diff
@@ -0,0 +1,234 @@
+#ifndef FASTSCAPELIB_DEALII_GRID_H_
+#define FASTSCAPELIB_DEALII_GRID_H_
+
+#include <unordered_map>
+#include <vector>
+
+#include <xtensor/xbroadcast.hpp>
+#include <xtensor/xmath.hpp>
+
+#include "fastscapelib/grid/base.hpp"
+#include "fastscapelib/utils/xtensor_utils.hpp"
```

Comment:
This file no longer appears to exist in the current fastscapelib repo: https://github.com/fastscape-lem/fastscapelib/tree/main/include/fastscapelib/utils How do we want to address this?